tests/lib.rs

```diff
@@ -88,7 +88,7 @@ struct Proto(Protocol<'static>);
 impl Arbitrary for Proto {
     fn arbitrary<G: Gen>(g: &mut G) -> Self {
         use Protocol::*;
-        match u8::arbitrary(g) % 26 {
+        match u8::arbitrary(g) % 27 {
             // TODO: Add Protocol::Quic
```

Comment:
Is this TODO still relevant?

Reply:
Don't think so, no. Removed with 7b8f4dd.